content/developer/reference/standard_modules/account/account_chart_template.rst

```diff
@@ -0,0 +1,62 @@
+.. _reference/account_chart_template:
+
+==============
+Chart Template
+==============
+
+---------------------
+Invoicing (`account`)
+---------------------
+
+This is the main module for Invoicing.
+
+.. autoclass:: odoo.addons.account.models.chart_template.AccountChartTemplate
+
+    .. autofield:: name
+    .. autofield:: parent_id
+    .. autofield:: code_digits
+    .. autofield:: visible
+    .. autofield:: currency_id
+    .. autofield:: country_id
+    .. autofield:: use_anglo_saxon
+    .. autofield:: bank_account_code_prefix
+    .. autofield:: cash_account_code_prefix
+    .. autofield:: transfer_account_code_prefix
```

Comment:
You lost my notes about those fields. The information from autofield is not enough

# Current version

![image](https://user-images.githubusercontent.com/186131/143893613-b754af7e-b35e-494b-8d61-98c10b7bec14.png)

# My version

![image](https://user-images.githubusercontent.com/186131/143893714-db77d74f-94a7-4d88-8974-71f6458ab2f2.png)

Reply:
https://github.com/odoo/documentation/pull/1301

Reply:
https://github.com/odoo/documentation/pull/1334#discussion_r759242928